### ISSUE TITLE
The

### ISSUE THREAD

**Issue by Xiaobaishushu25** [vizia/vizia#452]

**Comment by geom3trik** [comment]
Yes sorry, vizia is missing IME (Input Method Editor) at the moment. Hoping to add this soon.

**Comment by hipitt** [comment]
> Yes sorry, vizia is missing IME (Input Method Editor) at the moment. Hoping to add this soon.

Very much looking forward to it

**Comment by sout233** [comment]
I fixed it in #566
